weblate/static/loader-bootstrap.js

```diff
@@ -1546,6 +1546,22 @@ $(function () {
     $('#position-input').on('click', function() {
         $('#position-input').hide();
         $('#position-input-editable').show();
+        $('#position-input-editable input').focus();
+    });
+    $(document).on('click', function(event) {
```

Comment:
Do I read it correctly it catches all clicks on the page? Isn't that performance issue?

Reply:
A better approach would be to attach this event handler upon entering inline position editing, and then remove the handler upon leaving inline editing. Otherwise, users will have a click handler firing on every single click for the lifetime of a loaded page. The same concept applies for the `keyup` below.

For the actual checking to leave inline editing, something like this should work (didn't test but shows the concept):
```js
var editableInput = document.querySelector('#position-input-editable');
if (!editableInput.contains(event.target)) {
    // logic to leave inline editing
}
```
(Note how a test like `$('#position-input-editable').is(":visible"))` would be irrelevant when such an event handler is attached only when editing inline)

Reply:
@nijel @julen valid point. I will change how the action listener is attached.